staff/marti-malek/gamma-template/style/style.sass

```diff
@@ -0,0 +1,701 @@
+%img
```

Comment:
cool use of inheritance! But in fact if you are not going to reuse the code, then it's not necessary.

Reply:
True, i've changed it!